tests/integration/django/bamboo/leaves/features/content.feature

```diff
@@ -62,6 +62,24 @@ Feature: Count number of emails sent
       Quantity: Many
       """
 
+  Scenario: HTML alternatives
+    Given I send a test email with the following set:
+      """
+      from_email: orders@bamboodirect.com
+      to:
+        - undisclosed recipients;
+      subject: Try b4mb00 now!
+      body: |
+        This message is not spam.
+      html: |
+        <blink>GET B4MB00 CHEAP NOW!</blink>
+      """
+
+    Then I have sent an email with the following HTML alternative:
+      """
+      <blink>GET B4MB00 CHEAP NOW!</blink>
```

Comment:
...or that you are _using_ `parse_html` instead of string comparison. Perhaps leave this the only test, but add whitespace or different casing to showcase the fact that HTML is, in fact, compared properly.